packages/core/database/src/index.ts

```diff
@@ -49,7 +52,7 @@ const afterCreate =
   };
 
 class Database {
-  connection: Knex;
+  #connection?: Knex;
```

Comment:
Maybe someday they will have consistency between # and ts private keyword 🙌

Reply:
Do we have a preference for Strapi?

Reply:
I would vote for # as its enforced by javascript itself

Reply:
private in ts is just a type check thing and # is actually runtime enforced so # for actual strictness is better. hopefully the behaviour will converge because I really dislike this syntax 😂